src/network/gossip.rs

```diff
@@ -47,7 +48,7 @@ const CONTACT_INFO: &str = "contact-info";
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Config {
     pub address: String,
-    pub announce_address: String,
+    pub announce_address: Option<String>,
```

Comment:
Can you leave this as a `String` and can we have empty string mean unspecified? In our config, we define envvars for config parameters that are different by host and use that in the config. It's more annoying to omit a config parameter than to set the envvar for it to empty string.

Reply:
Addressed! https://github.com/farcasterxyz/snapchain/pull/402/commits/5641a6c019b50324da2946c46a17a03ea1b9dcba